### PR TITLE
Fix errors in README

### PR DESCRIPTION
**Pull request by aleksasteam**
- Fix the broken link in header
- Fix the example C++ code (putting ':' tells Qt to look in the root of the resource directory)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 [![Latest PyPI version](https://img.shields.io/pypi/v/QDarkStyle.svg)](https://pypi.python.org/pypi/QDarkStyle)
 [![License: MIT](https://img.shields.io/dub/l/vibe-d.svg)](https://opensource.org/licenses/MIT)
 [![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by/4.0/)
-[![conduct](https://img.shields.io/badge/code%20of%20conduct-contributor%20covenant-green.svg?style=flat-square)][http://contributor-covenant.org/version/1/4/]
+[![conduct](https://img.shields.io/badge/code%20of%20conduct-contributor%20covenant-green.svg?style=flat-square)](http://contributor-covenant.org/version/1/4/)
```
Comment (dpizetta) [inline README.md]: Thanks!

File: README.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ From code: Download/clone the project, go to ``qdarkstyle`` folder then:
 - Load the stylesheet:
 
   ```cpp
-  QFile f("qdarkstyle/style.qss");
+  QFile f(":qdarkstyle/style.qss");
```
[the remark this answers is not in the document]
Comment (aleksasteam) [reply README.md]: IIRC I added it as a resource, and for it to work I had to add ':'. See [here](http://doc.qt.io/qt-5/resources.html#using-resources-in-the-application). It's for accessing resources, if you are accessing a real file, as you said, you don't need it.

I thought that the second step in the README meant adding the file as a resource, and I needed to add the ':' for it to work that way. Thought it was missing, so I sent a PR :blush:

[the remark this answers is not in the document]
Comment (ColinDuquesnoy) [reply README.md]: @aleksasteam You're right. The second step in the README meant adding the file as a resource. The missing ``:`` is a typo and is really needed.

@dpizetta FYI the ``:`` is not specific to C++, we use it to load the stylesheet with python [as well](https://github.com/ColinDuquesnoy/QDarkStyleSheet/blob/master/qdarkstyle/__init__.py#L185).

[the remark this answers is not in the document]
Comment (dpizetta) [reply README.md]: @aleksasteam thanks again! @ColinDuquesnoy thanks for the information.

[the remark this answers is not in the document]
Comment (dpizetta) [reply README.md]: Some references for the registry: 
http://thisthread.blogspot.com.br/2011/04/accessing-file-as-resource.html
http://doc.qt.io/archives/vs-addin/vs-addin-managing-resources.html
http://doc.qt.io/qt-5/resources.html

**Comment by lincolningaroca** [comment]
Hi guys, I had a problem applying the css, it happens with the QComboBox, when deploying the combo, it looks very bad, the combo size is distorted, as invisible items would have been added.

but this happens only when the fusion style is applied, otherwise it looks normal.

here is a screenshot.

image

image

**Comment by dpizetta** [comment]
@lincolningaroca  Could you open a new issue for that and add info about your system asked in the issue report? Also, it is very important that you remove names and e-mails from other people from the image you post here (if it is not a bummy DB). Tks

**Pull request merged**